misc/apply-cache-diff.py

```diff
@@ -20,8 +20,8 @@
 def make_cache(input_dir: str, sqlite: bool) -> MetadataStore:
     if sqlite:
         return SqliteMetadataStore(input_dir)
-    else:
-        return FilesystemMetadataStore(input_dir)
+
```

Comment:
Not a fan of these changes as they reduce symmetry.

Reply:
Yes, might be a good modification to the rule to avoid rewriting into an early-return where the two parts are somewhat symmetrical in number of statements, or maybe simply when they're short.